db/blip_handler.go

```diff
@@ -478,8 +478,18 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, opts *sendChangesOptions
 					}
 
 				}
-				for _, item := range change.Changes {
-					changeRow := bh.buildChangesRow(change, item["rev"])
+				// if V3 and below populate change row with rev id
```

Comment:
Should move the common bit down out of the if so we're not duplicating the same work for different protocol versions.

```go
				// if V3 and below populate change row with rev id
				if bh.activeCBMobileSubprotocol <= CBMobileReplicationV3 {
					for _, item := range change.Changes {
						changeRow := bh.buildChangesRow(change, item["rev"])
						pendingChanges = append(pendingChanges, changeRow)
					}
				} else {
					changeRow := bh.buildChangesRow(change, change.CurrentVersion.String())
					pendingChanges = append(pendingChanges, changeRow)
				}

				if err := sendPendingChangesAt(opts.batchSize); err != nil {
					return err
				}
```